packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -133,12 +134,12 @@ export function getMomentTemplateBanner(
                                 css={
                                     isUsEoyGivingTuesBanner
                                         ? [
-                                              styles.desktopVisualContainer,
+                                              styles.desktopVisualContainer(isNewYearBanner),
```

Comment:
Same in terms of composition here

Reply:
Yep, understood remove parameter to use 2 simpler css parts. All done.